packages/react-native/Libraries/Components/TextInput/TextInput.d.ts

```diff
@@ -150,6 +150,11 @@ export interface TextInputIOSProps {
    */
   clearTextOnFocus?: boolean | undefined;
 
+  /**
+   * If true, the placeholder will not be scaled down to fit the text input
+   */
+  disablePlaceholderScaling?: boolean | undefined;
+
```

Comment:
You're missing the type definition in `.flow` file https://github.com/facebook/react-native/blob/fdfa0b17bb8c0b55c9c2569e7d3ffcf11c0c5072/packages/react-native/Libraries/Components/TextInput/TextInput.flow.js